group/secretsharing/poly.go

```diff
@@ -0,0 +1,73 @@
+package secretsharing
+
+import (
+	"errors"
+	"io"
+
+	"github.com/cloudflare/circl/group"
+)
+
+type polynomial struct {
+	deg   uint64
+	coeff []group.Scalar
```

Comment:
Document that this represents the polynomial  `\sum sum_i  coeff[i] x^i`. (You might've flipped the order.)